get_info.php

```diff
@@ -168,7 +177,7 @@ function handle_get($params)
             }
             $user_id = $params["user_id"];
             $target_ta = $params["target_ta"];
-            $course_code = array_get($params, "course");
+            $course_code = array_get($params, "course_code");
```

Comment:
This is wrong. The API specifies `"course"`

Reply:
There is a inconsistency in the API documentation, I assumed it to be `course_code` for everything. Thus all API requests have `course_code` in the front-end. Could I change the API specification to be `course_code`?

Reply:
You may. You will also need to change the test that uses `"course"`